lib/web_ui/lib/src/engine/canvaskit/renderer.dart

```diff
@@ -384,47 +353,76 @@ class CanvasKitRenderer implements Renderer {
     frameTimingsOnBuildFinish();
     frameTimingsOnRasterStart();
 
+    // TODO(harryterkelsen): Create and dispose the Rasterizer when the views
+    // are registered/disposed.
```

Comment:
`EngineFlutterView.dispose()` is already available. Is it worth leaving this as a TODO? Also, TODOs require a Github issue attached to them